qiskit_ibm_runtime/api/rest/runtime_session.py

```diff
@@ -35,11 +37,23 @@ def __init__(self, session: RetrySession, session_id: str, url_prefix: str = "")
         """
         super().__init__(session, "{}/sessions/{}".format(url_prefix, session_id))
 
-    def close(self) -> None:
-        """Close this session."""
+    def cancel(self) -> None:
+        """Cancel all jobs in the session."""
         url = self.get_url("close")
         self.session.delete(url)
 
+    def close(self) -> None:
+        """Set accepting_jobs flag to false, so no more jobs can be submitted."""
+        payload = {"accepting_jobs": False}
+        url = self.get_url("self")
+        try:
+            self.session.patch(url, json=payload)
+        except RequestsApiError as ex:
+            # TODO change this to 404
+            if ex.status_code == 500:
```

Comment:
PR should be good to go, just need to change this to `404` once the IQP API changes are in production